packages/manager/modules/ng-layout-helpers/src/list/list-layout.utils.js

```diff
@@ -1,5 +1,11 @@
 import get from 'lodash/get';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
I see imports from lodash-es and lodash. maybe its a good idea to stick to one of these.